.flowconfig

```diff
@@ -23,7 +23,11 @@
 .*/Libraries/polyfills/.*
 
 ; Ignore metro
-; .*/node_modules/metro/.*
+.*/node_modules/metro/.*
```

Comment:
How close does this get us to the CI loops passing? If we're still not close, there's always the risk these just break again without people realizing.